en/docs/install-and-setup/setup/setting-up-databases/data-dictionary/workflow-related-tables.md

```
@@ -0,0 +1,137 @@
+# Workflow Related Tables
+
+This section lists out all the workflow related tables and their attributes in the WSO2 API Manager database.
+
+---
+
+### AM_WORKFLOWS
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)